doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc

```diff
@@ -1,6 +1,16 @@
 [id='jws-jdbc-config_{context}']
 = Configuring JDBC Web Server data sources
-Java Database Connectivity (JDBC) is an API specification that is used to connect programs written in Java to the data in popular databases. A data source is an object that enables a JDBC client, such as an application server, to establish a connection with a database. Applications look up the data source on the Java Naming and Directory Interface (JNDI) tree or in the local application context and request a database connection to retrieve data. You must configure data sources for {KIE_SERVER} to ensure proper data exchange between the servers and the designated database.
+
+Java Database Connectivity (JDBC) is an API specification that is used to connect programs written in Java to the data in popular databases. A data source is an object that enables a Java Database Connectivity (JDBC) client, such as an application server, to establish a connection with a database. Applications look up the data source on the Java Naming and Directory Interface (JNDI) tree or in the local application context and request a database connection to retrieve data. You must configure data sources for {KIE_SERVER} to ensure correct data exchange between the servers and the designated database.
```

Comment:
I had to add a Cr above this text in the original file and github decided it was a new chunk of text, so I didn't actually work on it. Nevertheless, I will remove "popular" because you are right.

Reply:
Ah ok, understood.